core/services/relay/evm/evm.go

```diff
@@ -116,17 +117,16 @@ func (r *Relayer) Start(context.Context) error {
 }
 
 func (r *Relayer) Close() error {
-	return r.mercuryPool.Close()
+	return nil
 }
 
 // Ready does noop: always ready
 func (r *Relayer) Ready() error {
-	return r.mercuryPool.Ready()
+	return nil
 }
 
 func (r *Relayer) HealthReport() (report map[string]error) {
 	report = make(map[string]error)
-	services.CopyHealth(report, r.mercuryPool.HealthReport())
```

Comment:
Does this check need to be moved somewhere? The application maybe?

Reply:
If it is in the top level slice of application services, then it will be handled automatically.